File: rohitMigrationTesting/pipelines/pipeline2/code/src/main/scala/io/prophecy/pipelines/pipeline1/graph/Reformat_1.scala
package io.prophecy.pipelines.pipeline1.graph

import io.prophecy.libs._
import io.prophecy.pipelines.pipeline1.config.ConfigStore._
import io.prophecy.pipelines.pipeline1.config.Context
import io.prophecy.pipelines.pipeline1.udfs.UDFs._
import io.prophecy.pipelines.pipeline1.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._
object Reformat_1 { def apply(context: Context, in: DataFrame): DataFrame = in }

File: MigrationPy/pipelines/p2/code/p1/graph/Reformat_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from prophecy.transpiler import call_spark_fcn
from prophecy.transpiler.fixed_file_schema import *
from p1.config.ConfigStore import *
from p1.udfs.UDFs import *

def Reformat_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.select(lit(Config.pipearray_of_rec[0].arrayinsiderec[0].val3).alias("id"), col("first_name"))
